### PR TITLE
Delete mdbm lock files at startup

### PR DESCRIPTION
**Pull request by hmusum**

### CODE REVIEW

File: configserver/src/main/sh/start-configserver [hunk 1]
```diff
@@ -156,6 +156,10 @@ export LD_PRELOAD=${VESPA_HOME}/lib64/vespa/malloc/libvespamalloc.so
 printenv > $cfpfile
 fixddir $bundlecachedir
 
+# Delete mdbm lock files
+# TODO: Remove when scoreboard is no longer used
+$(ROOT)/bin64/mdbm_delete_lockfiles $(ROOT)/var/scoreboards/configserver.sbm || true
```
Comment (jobergum) [inline configserver/src/main/sh/start-configserver]: Is mdbm* available on all distributions?

Comment (hmusum) [reply configserver/src/main/sh/start-configserver]: No, but we handle that the command fails. 

This cleanup is not necessary on all distributions, but I am not sure how we can check where this should be done, at least not when it is done in a start script.

Comment (baldersheim) [reply configserver/src/main/sh/start-configserver]: Don't we have a yahoo specific startfile ?
We could also do it inside the container.
This should also be done on all container nodes. There has also been tenant issues related to this.

**Comment by hmusum** [comment]
Will implement in another place instead

**Pull request closed without merging**